### PR TITLE
Enable Ubuntu arm crossgen_comparison Checked in each PR

### PR DESCRIPTION
**Pull request by echesakovMSFT**
Merge after #20137

### CODE REVIEW

**Comment by echesakovMSFT** [comment]
@dotnet-bot test ci please

File: netci.groovy [hunk 1]
```diff
@@ -1956,6 +1956,9 @@ def static addTriggers(def job, def branch, def isPR, def architecture, def os,
                                 isDefaultTrigger = true
                             }
                             break
+                         case 'crossgen_comparison':
+                            isDefaultTrigger = shouldGenerateCrossGenComparisonJob(os, architecture, configuration, scenario)
```
Comment (BruceForstall) [inline netci.groovy]: I don't think we should do both Checked and Release for every PR. I think that's too much. I think we should do just Checked. Then, do Release as a non-PR `@daily` triggered job.

**Comment by echesakovMSFT** [comment]
@dotnet-bot test ci please

File: netci.groovy (hunk 1 above)
Comment (echesakovMSFT) [reply netci.groovy]: Did as we discussed:
1. only **Ubuntu arm Checked** job runs by default
2. both **Ubuntu arm Checked and Release** will run daily

**Pull request merged**